lib/coverband/collectors/coverage.rb

```diff
@@ -10,6 +10,10 @@ module Collectors
     # and this method is currently in a ensure block in middleware and threads
     ###
     class Coverage
+      RUNTIME_TYPE = nil
```

Comment:
This is a great change as it makes the code more readable. I felt a little funny about making this nil honestly but it gave backwards compatibility to existing data so felt it was worth it.

Reply:
yeah, I have been considering this... I might actually change it to `:run_time` and now that we have built migrations, have the migration upgrade the `nil` stored value to `runtime`... can't decide entirely. I was going to get everything working and then rethink the value of migrations etc.